File: PROGRAMMERS/lv0/2차원으로 만들기.js
function solution(num_list, n) {
    let ans = [];
    for (let i = 0; i < num_list.length; i += n) {
        ans.push(num_list.slice(i, i + n));
    }
    return ans;
}

let inps = [[[1, 2, 3, 4, 5, 6, 7, 8], 2]];

for (let i of inps) {
    console.log(solution(i[0], i[1]));
}

File: PROGRAMMERS/lv0/python/2차원으로 만들기.py
def solution(num_list, n):
    ans = []

    for v in range(0, len(num_list), n):
        ans.append(num_list[v:v+n])

    return ans

inps = [
    [[1, 2, 3, 4, 5, 6, 7, 8], 2],
    [[100, 95, 2, 4, 5, 6, 18, 33, 948], 3]
]

for (nl, n) in inps:
    print(solution(nl, n))